shared/output/gapic/templates/showcase/google-showcase.gemspec

```diff
@@ -0,0 +1,32 @@
+# -*- ruby -*-
+# encoding: utf-8
+
+require File.expand_path("lib/google/showcase/version", __dir__)
+
+Gem::Specification.new do |gem|
+  gem.name          = "google-showcase"
+  gem.version       = Google::Showcase::VERSION
+
+  gem.authors       = ["Google LLC"]
+  gem.email         = "googleapis-packages@google.com"
+  gem.description   = "google-showcase is the official library for Google Showcase API."
+  gem.summary       = "API Client library for Google Showcase API"
+  gem.homepage      = "https://github.com/googleapis/googleapis"
+  gem.license       = "Apache-2.0"
+
+  gem.platform      = Gem::Platform::RUBY
+
+  gem.files         = `git ls-files -- lib/*`.split("\n") +
+                      ["README.md", "LICENSE", ".yardopts"]
+  gem.require_paths = ["lib"]
+
+  gem.required_ruby_version = ">= 2.0.0"
+
+  gem.add_dependency "google-gax", "~> 2.0"
+
+  gem.add_development_dependency "google-style", "~> 0.2"
```

Comment:
@dazuma has told us multiple times that he wants this to be the style guide for the generator.

Reply:
I would think google-style can be omitted from the gapic generator, but definitely included in the cloud generator.

Reply:
Yup, I was thinking the same as @quartzmo. For sure, it makes sense for the cloud generator.

For the base case I'm not against it if you think it would be welcomed by a typical Rubyist. I don't know enough about `google-style` to have an opinion either way as I've never used it.

Reply:
I'm thumbs down on "welcomed by a typical Rubyist" but very much thumbs up for adoption in Google's open-source Ruby packages.

Reply:
Perhaps @dazuma can weigh in here, since he has made a point to have the default be google-style several times.

Reply:
Everything we've done has been using the Google adopted style. The generator code as well as the generated code is matching that style. It would be a considerable effort to change it, but I also think it is a sensible default for anything in the googleapis repo. (IIRC, my original opinion was to use vanilla Rubocop for the default generator, and google-style for the cloud generator. But from previous discussions about this I've been encouraged to change my mind, which I have.)

I think the extensible nature of the ruby generator gives users enough control over the output that we can keep this as is. We have many ways for users to control the format their generated code. They can either set a `format_config` CLI option pointing to the rubocop rules they prefer to use, or they can generate a custom generator gem and change the default `format_config` file to be used, or they can generate a custom generator gem and change the formatter from Rubocop to something completely different like Rubyfmt.

Reply:
👍 Sounds good! Yes, let's avoid the rework of changing the rules in gapic-generator.

Reply:
Sorry I totally missed this conversation. Yes, I would like for google-style to be the default even for the base generator. The intent is to infiltrate the style into as many google properties as possible, as the default, so that the "Google" brand presents a unified face to the community. If a particular sub-generator wants to change the style, it should be easy, but the the default should be our preferred style.